pkg/client/config.go

```diff
@@ -26,3 +28,30 @@ func ConfigFromEnv() (*rest.Config, error) {
 		},
 	}, nil
 }
+
+func LoadConfig(configFileFrom, configFileName, configContext string) (*rest.Config, error) {
+	var config *rest.Config
+	var err error
+
+	switch configFileFrom {
+	case "in-cluster":
+		config, err = rest.InClusterConfig()
+	case "environment":
+		config, err = ConfigFromEnv()
+	case "file":
```

Comment:
Is there a readme that can be updated to reference how to start smith?

Reply:
No, this is not documented (yet).